diff -up dwm-6.0-original/config.h dwm-6.0-custom/config.h
--- dwm-6.0-original/config.h	2012-07-08 01:37:39.533236414 -0600
+++ dwm-6.0-custom/config.h	2012-07-08 02:52:21.814726694 -0600
@@ -30,8 +30,6 @@ static const Bool resizehints = True; /*
 static const Layout layouts[] = {
 	/* symbol     arrange function */
 	{ "[]=",      tile },    /* first entry is default */
-	{ "><>",      NULL },    /* no layout function means floating behavior */
-	{ "[M]",      monocle },
 };
 
 /* key definitions */
