diff -up dwm-6.0-custom/config.h dwm-6.0-modified/config.h
--- dwm-6.0-custom/config.h	2012-07-08 04:06:42.594683406 -0600
+++ dwm-6.0-modified/config.h	2012-07-08 04:07:07.568014759 -0600
@@ -12,7 +12,7 @@ static const unsigned int borderpx  = 1;
 static const unsigned int snap      = 32;       /* snap pixel */
 static const unsigned int systrayspacing = 2;   /* systray spacing */
 static const Bool showsystray       = True;     /* False means no systray */
-static const Bool showbar           = True;     /* False means no bar */
+static const Bool showbar           = False;     /* False means no bar */
 static const Bool topbar            = True;     /* False means bottom bar */
 static const Bool viewontag         = True;     /* Switch view on tag switch */
 
